apps/aechannel/src/aesc_client_reconnect_tx.erl

```diff
@@ -0,0 +1,185 @@
+-module(aesc_client_reconnect_tx).
+
+-behavior(aetx).
+-behavior(aesc_signable_transaction).
```

Comment:
There should be a comment saying that this TX needs to be signed by the person reconnecting in order to prove the identity of the peer.

Reply:
Now explained in the top-level comment about the module.